actinia-ubuntu/Dockerfile

```diff
@@ -1,48 +1,43 @@
-FROM osgeo/grass-gis:stable-ubuntu
+FROM osgeo/grass-gis:stable-ubuntu as grass
```

Comment:
```suggestion
FROM osgeo/grass-gis:current-ubuntu as grass
```

Reply:
I do have some changes I have not pushed yet (as some stuff was failing).
The SNAP stuff could be more separated. Unfortunately, snappy is still not possible to install with Python 3.10.
It will be renamed to esa_snappy and is probably first usable with SNAP 10 (just guessing). Will push an update soon...